xml/System.Configuration/SectionInformation.xml

```diff
@@ -1107,6 +1107,9 @@
 > [!NOTE]
 >  If you call the <xref:System.Configuration.SectionInformation.ProtectSection%2A> method with a `null` parameter or an empty string, the <xref:System.Configuration.RsaProtectedConfigurationProvider> class is used as the protection provider.  
   
+> [!NOTE]
+>  If you call the <xref:System.Configuration.SectionInformation.ProtectSection%2A> method on the Linux operating system, a <xref:System.PlatformNotSupportedException> will be thrown when attempting to save the configuration.
```

Comment:
```suggestion
>  If you call the <xref:System.Configuration.SectionInformation.ProtectSection%2A> method on the Linux operating system, a <xref:System.PlatformNotSupportedException> will be thrown when you attempt to save the configuration by calling <xref:System.Configuration.Configuration.Save%2A
```

Also, I don't think this needs to be in a note block. It's already in the Remarks section, which I think makes sense.